app/components/system-status/component.js

```diff
@@ -0,0 +1,46 @@
+/* jshint ignore:start */
+
+import Ember from 'ember';
+import ENV from 'irene/config/environment';
+import {isNotFoundError} from 'ember-ajax/errors';
+
+export default Ember.Component.extend({
+
+  isStorageWorking: false,
+  isDeviceFarmWorking: false,
+
+  didInsertElement() {
```

Comment:
Move this to didRender instead.. didInsertElement is usually used to access and hook into dom.